### PR TITLE
vvv331 issue fixes

### PR DESCRIPTION
**Pull request by VoiditeDev**

### CODE REVIEW

File: src/init.luau [hunk 1]
```diff
@@ -1,6 +1,6 @@
 --[[
 	A module to work with detecting devices.
-	Programmed by break-core.
+	Programmed by break-core. Contributed by VSCPlays
```
Comment (break-core) [inline src/init.luau]: I don't think there's a reason to have this here. Your GitHub name will show up on the contributors list once this goes through.

Comment (break-core) [reply src/init.luau]: I might be a little biased on this though... but it is kind of redundant to have this here.

Comment (VoiditeDev) [reply src/init.luau]: removed it

**Comment by break-core** [comment]
Hi! Sorry for being late. I slept for a while.

I'll merge your PR now. LGTM!

**Pull request merged**